epsagon/wrappers/aiohttp.py

```diff
@@ -38,16 +41,24 @@ async def AiohttpMiddleware(request, handler):
 
     try:
         body = await request.text()
+        print_debug('[aiohttp] got body')
         runner = AiohttpRunner(time.time(), request, body, handler)
         trace.set_runner(runner)
         collect_container_metadata(runner.resource['metadata'])
+        print_debug('[aiohttp] initialized runner')
     except Exception as exception: # pylint: disable=W0703
         warnings.warn('Could not extract request', EpsagonWarning)
 
     raised_err = None
     try:
         response = await handler(request)
+        print_debug('[aiohttp] got response')
     except Exception as exception:  # pylint: disable=W0703
+        # Ignoring 404s
+        if type(exception).__name__ == 'HTTPNotFound':
```

Comment:
can we import the specific exception class and compare the types objects?

Reply:
<3